docs/source/get-started.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Get started
+description: Learn how to quickly set up Apollo Client in your Android app
+---
+
+Apollo-Android is a GraphQL compliant client that generates Java models from standard GraphQL queries.  These models give you a typesafe API to work with GraphQL servers. Apollo will help you keep your GraphQL query statements together, organized, and easily accessible from Java.
+
+When you change a query and recompile your project - Apollo code generation will rebuild your data model for you. Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection. Future versions of Apollo-Android will also work with AutoValue and other value object generators.
```

Comment:
Maybe `Apollo code generator`

Reply:
Take out part about Autovalue  please that was unfortunately is not in current plans